framework/source/class/qx/ui/form/Spinner.js

```diff
@@ -118,6 +118,17 @@ qx.Class.define("qx.ui.form.Spinner",
     } else {
       this.initValue();
     }
+
+    // forward the focusin and focusout events to the textfield. The textfield
+    // is not focusable so the events need to be forwarded manually.
+    this.addListener("focusin", function(e) {
+      textField.fireNonBubblingEvent("focusin", qx.event.type.Focus);
+      textField.selectAllText();
```

Comment:
I personally would prefer a "cursor after text" in all textfield related widgets.